libr/include/r_anal.h

```diff
@@ -282,6 +282,7 @@ typedef struct r_anal_function_t {
 	ut64 addr;
 	RPVector vars;
 	HtUP/*<st64, RPVector<RAnalVar *>>*/ *inst_vars; // offset of instructions => the variables they access
+	int bp_off;
```

Comment:
Please describe the exact semantics of this field, also with a comment. Also I am not sure whether `int` is a good choice here. `st64` is probably better.